### PR TITLE
Skeleton arrow pickup and username formatting fix

### LINKED ISSUE #37: Skeleton undeads can farm arrows/bows
### Expected behavior

- Skeleton bows/arrow shouldn't be dropped on death.
- Arrows shouldn't be able to be picked up from skeleton undead's bow shots

### Actual behavior

- Skeleton bows/arrow drop on each death
- Arrows can be picked up if it's stuck in the ground when shot by a skeleton undead

### Steps to reproduce

-
1. Become skeleton undead
2. Die
3. Go to death location
4. Pick up bow

-
1. Become skeleton undead
2. Shoot an arrow
3. Pick up arrow

### Minecraft version and OS

1.18.1, Windows 10

### Checks

- [X] I am using modifications (mods, resourcepacks, etc...). (check if yes)
- [X] I asked in the Discord to see whether this issue is in fact a bug that needs to be fixed.

### Anything else

_No response_

### LINKED ISSUE #41: Incorrect username formatting in #smp-chat
### Expected behavior

Names containing underscores or other special characters should not be formatted in the discord message.

### Actual behavior

Names with underscores or other special characters in some cases are formatted in the discord message. Attached is an example.
Screenshot 2022-02-01 at 14 07 12

### Steps to reproduce

Have anyone with underscores in their name (aka Bag_of_Doritos) join, leave, or chat in the server and observe #smp-chat.

### Minecraft version and OS

N/A

### Checks

- [ ] I am using modifications (mods, resourcepacks, etc...). (check if yes)
- [X] I asked in the Discord to see whether this issue is in fact a bug that needs to be fixed.

### Anything else

_No response_

### PR DESCRIPTION
**Pull request by Lianecx**
Arrows can still be picked up by peole in creative. 
fixed #37 
fixed #41

### CODE REVIEW

**Review by srnyx — APPROVED** [review]

**Review by srnyx — APPROVED** [review]

**Comment by Lianecx** [comment]
Ye ik i forgot to do that

**Comment by Lianecx** [comment]
i did it again -_-
Sorry ill create a new branch

**Comment by Lianecx** [comment]
Dont merge it yet, i think it escapes the emojis

**Comment by Lianecx** [comment]
Should chat messages also be escaped or should people be allowed to send stuff like **bold** to discord?

**Comment by srnyx** [comment]
> Should chat messages also be escaped or should people be allowed to send stuff like **bold** to discord?

They should be allowed in chat messages, only the name should be escaped.
Also, make sure that it doesn't escape the `**` around the name when a user joins/leaves the game.

**Comment by Lianecx** [comment]
Okay great, @commandgeek you can approve this now

File: src/main/java/com/commandgeek/GeekSMP/listeners/discord/DiscordMessageCreateListener.java [hunk 1]
```diff
@@ -89,7 +89,7 @@ public static boolean command(Message message) {
                     MuteManager.mute(player.getName(), null, null, sender);
                     new MessageManager("discord-mute-permanent")
                             .replace("%user%", user.getMentionTag())
-                            .replace("%player%", player.getName())
+                            .replace("%player%", player.getName(), true)
```
Comment (commandgeek) [inline src/main/java/com/commandgeek/GeekSMP/listeners/discord/DiscordMessageCreateListener.java:92]: Adding "true" to these placed doesn't make sense to me, and it is throwing errors. Is there a specific reason for this?

**Comment by Lianecx** [comment]
Checks passed, it also didnt throw any errors for me...
I overloaded the replace method to have a third argument which escapes the markdown of the replacment text and then replaces it

**Comment by srnyx** [comment]
> Checks passed, it also didnt throw any errors for me... I overloaded the replace method to have a third argument which escapes the markdown of the replacment text and then replaces it

Reply inside the conversation please

**Comment by Lianecx** [comment]
okay sorry

**Comment by Lianecx** [comment]
didnt notice it was a comment

File: src/main/java/com/commandgeek/GeekSMP/listeners/discord/DiscordMessageCreateListener.java (hunk 1 above)
Comment (Lianecx) [reply src/main/java/com/commandgeek/GeekSMP/listeners/discord/DiscordMessageCreateListener.java:92]: Checks passed, it also didnt throw any errors for me...
I overloaded the replace method to have a third argument which escapes the markdown of the replacment text and then replaces it

Comment (commandgeek) [reply src/main/java/com/commandgeek/GeekSMP/listeners/discord/DiscordMessageCreateListener.java:92]: You're right, my bad. I didn't notice the additional replace method you created.

**Review by commandgeek — APPROVED** [review]

**Pull request merged**